js/about/extensions.js

```diff
@@ -24,10 +24,20 @@ class ExtensionItem extends ImmutableComponent {
   constructor () {
     super()
     this.onContextMenu = this.onContextMenu.bind(this)
+    this.onInspect = this.onInspect.bind(this)
   }
   onContextMenu (e) {
     aboutActions.contextMenu(this.props.extension.toJS(), 'extensions', e)
   }
+  onInspect (e) {
+    const extensionId = this.props.extension.get('id')
+    chrome.tabs.query(
+      {currentWindow: true, active: true},
+      function (tabArray) {
+        chrome.ipcRenderer.send('load-url-requested', tabArray[0].id, 'chrome-extension://' + extensionId + '/_generated_background_page.html')
```

Comment:
I'm not sure why this ipc message even exists, but please use `appActions.loadURLRequested` instead. Sending ipc messages directly is deprecated. However, I don't think this will work correctly in either case because it will create a new background page instead of opening the existing one